services/core-api/app/__init__.py

```diff
@@ -117,13 +117,15 @@ class Healthcheck(Resource):
 
         def get_health(self):
             service = {
-                'database': False,
-                'cache': False,
-                'nris': False,
-                'docgen': False,
-                'docman': False
+                'database': True,
```

Comment:
@v-y-a-s @vyasworks made some updates to the health check method. This skips `healthchecks` to fail when running the application locally

Reply:
Looks good